README.md

```diff
@@ -1,46 +1,51 @@
-# Registry application that is used to correct a business (Incorporation Application).
+# Registry application that is used to correct a business (Incorporation Application)
```

Comment:
Future state, we can update the description to include Alterations and Change of Registration filings. 
Not required right now!